.github/workflows/_provenance.yaml

```diff
@@ -20,3 +28,25 @@ jobs:
       upload-tag-name: ${{ github.ref_name }}
       provenance-name: ${{ github.event.repository.name }}.intoto.jsonl
       continue-on-error: true
+
+  # Provenance job for all images manifests
+  # SLSA generator is a reusable workflow
+  # pull-request event is [not supported](https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/container#supported-triggers)
+  # runs-on option is [not supported](https://github.com/orgs/community/discussions/25783)
+  # ENV option is [not supported](https://github.com/orgs/community/discussions/26671)
+  # Reusable workflow doesn't support exrernal COSIGN_REPOSITORY via input / env variable
+  # TODO:
+  #   Split provenance jobs for internal / official releases when repositories are split
+  images-provenance:
+    strategy:
+      fail-fast: true
+      matrix:
+        IMAGE: ${{ fromJSON(inputs.images) }}
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@07e64b653f10a80b6510f4568f685f8b7b9ea830 #v1.9.0
```

Comment:
This needs to be a tag and use v1.10.0 for cosign to work without `invalid key` error against sig store TUF verification